sycl/test-e2e/Graph/RecordReplay/sub_graph.cpp

```diff
@@ -8,4 +8,4 @@
 
 #define GRAPH_E2E_RECORD_REPLAY
 
-#include "../Inputs/sub_graph.cpp"
\ No newline at end of file
+#include "../Inputs/sub_graph.cpp"
```

Comment:
This change looks unrelated to the proposed patch!

Reply:
I'm not quite sure why this is being highlighted as it's [unchanged from the current revision](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Graph/RecordReplay/sub_graph.cpp).

Reply:
I think this might be that a newline at EOF was added, but Github doesn't directly show them. In the old diff it indicates this was missing with the red warning icon.

Reply:
Ah right that makes sense, will just leave it as is then. Thanks Ben.